toolkit/scripts/pkggen.mk

```
@@ -110,7 +120,19 @@ $(graph_file): $(specs_file) $(go-grapher) $(toolchain_rpms)
 		$(if $(filter y,$(ENABLE_MEM_PROFILE)),--enable-mem-prof) \
 		$(if $(filter y,$(ENABLE_TRACE)),--enable-trace) \
 		--timestamp-file=$(TIMESTAMP_DIR)/grapher.jsonl \
-		--output $@
+		--output $@ \
+		$(if $(filter y,$(RESOLVE_CYCLES_FROM_UPSTREAM)), --resolve-cycles-from-upstream) \
+		$(if $(filter y,$(USE_PREVIEW_REPO)), --use-preview-repo) \
+		$(if $(filter y,$(IGNORE_VERSION_TO_RESOLVE_SELFDEP)), --ignore-version-to-resolve-selfdep) \
+		--output-dir=$(CACHED_RPMS_DIR)/cache \
+		--rpm-dir=$(RPMS_DIR) \
+		--toolchain-rpms-dir=$(TOOLCHAIN_RPMS_DIR) \
+		--toolchain-manifest=$(TOOLCHAIN_MANIFEST) \
+		--tls-cert=$(TLS_CERT) \
+		--tls-key=$(TLS_KEY) \
+		--tmp-dir=$(cache_working_dir) \
```

Do we want to use `grapher_working_dir` here?
```suggestion
		--tmp-dir=$(grapher_working_dir) \
```

---

Right! Thanks for noticing!